gradle/wrapper/gradle-wrapper.properties

```diff
@@ -17,6 +17,6 @@
 #Tue Aug 27 09:42:42 BST 2019
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.5-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.0.2-all.zip
```

Comment:
You will need to update the `gradlew` and `gradlew.bat` files.

https://docs.gradle.org/current/userguide/gradle_wrapper.html

Reply:
updated